### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by shdwy** [hop-protocol/hop-airdrop#364]
# Related Addresses

```
Chain1:
0x8ca4a304c2f55c4d9ccd41eeff70e1468c8f2422
0x94ea5b3c4b318a5fad55213cea0e600bb267a639
0x68adabaa3e031eacfcdfc1df3ca0d9990de44f39
0xd219ea014fe09b099ddfdcbd82d1b524d5cd0c54
0x84a4532d6398e75b36adbd204766f0a61fc9847b

Chain2:
0xf05c4c0beb5f2c89589e280ddb38a8a4dc4410ad
0x43bc98d884dc28dedf523758362f33153dc30b2c
0xba9213733a47d7604c791c5a77d020e29356c451
0xfb385af3e2c6b7f2aee9fef7de752d9c41008df8
0x5b9e3e1a2a12e85e12da784cdc900ced4b012eef
0x8a65eec7f9f55cea0d2b5c9386a582d744fbf5c1
0xcad36645facecbd1b3d872ecb86a82fcd09b7ce6
0xa10d011cbc17b3d7b9819b8f8ea5e0b171536030
0x62875c163652500678920f63673edf72badebb58
0xf4e748bda212c6d5e190a7ba59758a79beabd1eb
0x96f027d110cec6585eba6c14c0a8278b18ee7874
0x98e594b90b72ac612529b9422af1e747911e94f5
0x68d3426cc3863c25ca74e2121255022af9ca74f1
0x22875c65599a76090cb5e213da291b7f4f08acc9
0x9145f2d784fcb5e7895d5417df3db15ffe803cdf
0x578a011481262d559d3df6994905cd858fba0fd2

Chain3:
0xce117883aaa426956f4c04bfbb0dfdbfeb05fc43
0xc23b8da49d45144d10876c1fc52ebf1c4c6eff79
0xcc039a1ffa28143dec2e58809f1c319c92df659b
0x030d093893d9a592144458f5f5e54656dba575f7
0xaa959e868da555cbe5e85bbec6a7f1891f588a6e
0x8ff9b077cbab70eb02cd9f42abc2abe997df8321
0x0369d50f050ef548e2a601cd99f313a6d76cdc34
0x9af442db3c97d9525242d47f41454387a19f01a4
0x04b9b73e774493b3d28db74e698d848d901843a8
0x25ed8a6cf51a40ecb3025acff40494667b683002
0xcac0b640c75c5629a57a388d294226e6fd9666f3

Chain4:
0xf59d5eb13f730fe5675d76cbbd32f9e7f8146d5a
0x8a60264f0d213b585c684f2bd34c8f9e54a666ab
0xb11d6caea39f75f7103e642f384bbb7fd26c56c7
0x6785885a7616b65e26d0d8e2ea772f1daee196b1
0x5a66b866d65e177d565687fac04545925cddb0a5
0x30d696354740c18ad3b3968690120f7ea2e09dc7
0x48bc3c4f6a0867a595288071b5b35e343e4c6dfe
0xc9981f1a46a5c6cc5a5db4d77bb4bb94e7a66294
0x9338fbec9918831fa4fac26b6e0141ff610eb26b

Outliers
0xe25c50bc085061f60b972a1b6f6364636ac62562
0xff2d6dcd8771d988428868e264a5432ee4db3908
```

Links between chains:
 [Chain 1 & Chain 2](https://blockscout.com/xdai/mainnet/tx/0xaf7e9385a8fe60eae1c46707111aca4d9330611befd8de5cc0a1ad703430ce5f)
[Chain 2 & Chain 3](https://blockscout.com/xdai/mainnet/tx/0xe61aa62494e28718eb2f92d41d64e0b5684924f97511207f66adfb47904efeaf)
[Chain 2 & Chain 4](https://blockscout.com/xdai/mainnet/tx/0x619fe6750e4cc860f748c836e5a12e53dce8a81925b4e06ab50b16c5ca0fa426)

# Reasoning

All addresses receive funds from the previous address on the chain, they interact with the same set of contracts in the same order (in the same day) and then transfer funds to the next address on the chain.

# Methodology

I used blockscout API to figure out which address originally funded all 30k address eligible for the airdrop. Then I used excel to try and find chains of addresses that funded each other. Once I found a chain, I manually checked their activity on blockscout to analyze their behaviour and to look for links between chains.

The excel file is available in case you need it.

# Rewards Address

0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by shanefontaine** [comment]
Thank you for your report @shdwy. We have verified that the addresses in this report are Sybil attackers.

The report included 20 eligible addresses as Sybil attackers which means you are eligible for 2919.738533629433120345 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).

It looks like many of the addresses you posted were reported successfully in #257.

The qualified addresses are as follows:

```
0x8ca4a304c2f55c4d9ccd41eeff70e1468c8f2422
0x94ea5b3c4b318a5fad55213cea0e600bb267a639
0x68adabaa3e031eacfcdfc1df3ca0d9990de44f39
0xd219ea014fe09b099ddfdcbd82d1b524d5cd0c54
0x62875c163652500678920f63673edf72badebb58
0xf4e748bda212c6d5e190a7ba59758a79beabd1eb
0xc23b8da49d45144d10876c1fc52ebf1c4c6eff79
0xcc039a1ffa28143dec2e58809f1c319c92df659b
0x030d093893d9a592144458f5f5e54656dba575f7
0xaa959e868da555cbe5e85bbec6a7f1891f588a6e
0x8ff9b077cbab70eb02cd9f42abc2abe997df8321
0x0369d50f050ef548e2a601cd99f313a6d76cdc34
0x9af442db3c97d9525242d47f41454387a19f01a4
0x04b9b73e774493b3d28db74e698d848d901843a8
0x25ed8a6cf51a40ecb3025acff40494667b683002
0xcac0b640c75c5629a57a388d294226e6fd9666f3
0xc9981f1a46a5c6cc5a5db4d77bb4bb94e7a66294
0x9338fbec9918831fa4fac26b6e0141ff610eb26b
0xe25c50bc085061f60b972a1b6f6364636ac62562
0xff2d6dcd8771d988428868e264a5432ee4db3908
```

**Comment by shanefontaine** [comment]
In addition to the data you provided, some additional proof is as follows:

* Their behavior on FTM is nearly identical
* It looks like all addresses had similar behaviors on the Hop protocol


address | ensName | total | mainnet | arbitrum | optimism | polygon | xdai | totalVolume
-- | -- | -- | -- | -- | -- | -- | -- | --
0xcac0b640c75c5629a57a388d294226e6fd9666f3 |   | 35 | 0 | 0 | 0 | 18 | 17 | $3,341.85
0xc23b8da49d45144d10876c1fc52ebf1c4c6eff79 |   | 22 | 0 | 0 | 0 | 11 | 11 | $2,628.91
0xc9981f1a46a5c6cc5a5db4d77bb4bb94e7a66294 |   | 24 | 0 | 0 | 0 | 12 | 12 | $2,622.65
0x25ed8a6cf51a40ecb3025acff40494667b683002 |   | 16 | 0 | 0 | 0 | 7 | 9 | $2,199.84
0x8ca4a304c2f55c4d9ccd41eeff70e1468c8f2422 |   | 20 | 0 | 0 | 0 | 10 | 10 | $2,186.86
0xf4e748bda212c6d5e190a7ba59758a79beabd1eb |   | 20 | 0 | 0 | 0 | 10 | 10 | $2,089.83
0xff2d6dcd8771d988428868e264a5432ee4db3908 |   | 18 | 0 | 0 | 0 | 9 | 9 | $1,994.94
0x62875c163652500678920f63673edf72badebb58 |   | 21 | 0 | 0 | 0 | 11 | 10 | $1,994.30
0xe25c50bc085061f60b972a1b6f6364636ac62562 |   | 18 | 0 | 0 | 0 | 9 | 9 | $1,993.82
0x9338fbec9918831fa4fac26b6e0141ff610eb26b |   | 18 | 0 | 0 | 0 | 9 | 9 | $1,965.18
0x94ea5b3c4b318a5fad55213cea0e600bb267a639 |   | 18 | 0 | 0 | 0 | 9 | 9 | $1,924.35
0x68adabaa3e031eacfcdfc1df3ca0d9990de44f39 |   | 18 | 0 | 0 | 0 | 9 | 9 | $1,845.23
0xd219ea014fe09b099ddfdcbd82d1b524d5cd0c54 |   | 16 | 0 | 0 | 0 | 8 | 8 | $1,624.78
0x0369d50f050ef548e2a601cd99f313a6d76cdc34 |   | 14 | 0 | 0 | 0 | 7 | 7 | $1,605.97
0x9af442db3c97d9525242d47f41454387a19f01a4 |   | 14 | 0 | 0 | 0 | 7 | 7 | $1,593.59
0xcc039a1ffa28143dec2e58809f1c319c92df659b |   | 21 | 0 | 0 | 0 | 10 | 11 | $1,370.82
0xaa959e868da555cbe5e85bbec6a7f1891f588a6e |   | 13 | 0 | 0 | 0 | 7 | 6 | $1,368.32
0x8ff9b077cbab70eb02cd9f42abc2abe997df8321 |   | 12 | 0 | 0 | 0 | 6 | 6 | $1,335.63
0x030d093893d9a592144458f5f5e54656dba575f7 |   | 12 | 0 | 0 | 0 | 6 | 6 | $1,329.21
0x04b9b73e774493b3d28db74e698d848d901843a8 |   | 11 | 0 | 0 | 0 | 5 | 6 | $1,266.39

**Comment by 666erxia** [comment]
举报狗死全家族。每天都有家人死去，不得好死 这 2919.738533629433120345 HOP！送给你们买棺材

**Comment by 666erxia** [comment]
Family members die every day, and it is not easy to die 2919.738533629433120345 HOP! buy you a coffin

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly? This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.
0xD47CB1835EB36e4A108C33E446449192426Fc160 @shanefontaine
